README.md

```diff
@@ -1,2 +1,69 @@
-# module-ballerina-random
 Ballerina Random Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-random/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-random/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-random.svg)](https://github.com/ballerina-platform/module-ballerina-random/commits/main)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/random.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Frandom)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+The Random library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This module provides functions related to random number generation.
+
+## Issues and Projects
+
+Issues and Projects tabs are disabled for this repository as this is part of the Ballerina Standard Library. To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+This repository only contains the source code for the module.
```

Comment:
```suggestion
This repository contains only the source code of the module.
```
Shall we put this sentence at the beginning of the "## Issues and Projects" topic?